### DISCUSSION TITLE
No longer able to paste a cURL request into an existing request's URL textbox

### DISCUSSION THREAD

**Discussion by aibars** [Kong/insomnia#6346] [Feedback]
Hi everyone! I've been using Insomnia for several years almost everyday at work. I noticed that recently the ability to "import" a curl request by pasting directly to a URL box is no longer working. Instead, users need to create a new request and select the option "from curl" to achieve this, which requires creating a "new request" every time. Why has this feature changed? Is there a setting to enable curl parsing in the url textbox?

Thanks!

**Comment by vxsander** [comment]
I really miss this feature as well. The fromCurl option is nice but disabling the curl parsing feature in the url box removes one of the nicest features insomnia has. 😢

**Comment by jackkav** [comment]
re-introduced with preview and validation in #6384
